File: bin/Debug/scripts/convert_copied_paths.js
'use strict';

// Read input

process.stdin.resume();
process.stdin.setEncoding('utf8');

let inputData = "";
process.stdin.on('data', (inputLine) => {
    inputData += inputLine;
});

process.stdin.on('end', () => {
    let inputDataBuffer = Buffer.from(inputData, 'base64');
    let input = inputDataBuffer.toString('utf-8');

    main(input);
});

function main(input) {
    // Process

    input = input.replace(/\r\n/g, "\n");
    let lines = input.trim().split("\n");

    let output = "";
    lines.forEach(line => {
        output += line.replace(/^[\"]+|[\"]+$/g, "").replace(/\\/g, "/") + "\r\n";
    });

    if (!input.endsWith("\n")) {
        output = output.trimEnd();
    }

    // Output result

    console.log(Buffer.from(output).toString('base64'));
}

File: bin/Debug/scripts/convert_copied_paths.py
import sys
import base64

# encodingType = "base64_utf8"
encodingType = "systemDefault"

# 读取输入

if encodingType == "base64_utf8":
    inputText = base64.b64decode(sys.stdin.read()).decode('utf-8')
else:
    inputText = sys.stdin.read()

# 开始处理

str = inputText.replace("\r\n", "\n")
lines = str.strip().split("\n")

outputText = ""
for line in lines:
    outputText += line.strip('"').replace("\\", "/") + "\r\n"

# 输出结果

if encodingType == "base64_utf8":
    print(base64.b64encode(outputText.encode('utf-8')).decode('ascii'))
else:
    print(outputText.replace("\r\n", "\n"), end = '')
